security/guard_authentication.rst

```diff
@@ -513,3 +513,4 @@ Frequently Asked Questions
 
 .. _`Social Authentication`: https://github.com/knpuniversity/oauth2-client-bundle#authenticating-with-guard
 .. _`HWIOAuthBundle`: https://github.com/hwi/HWIOAuthBundle
+.. _`KnpUOAuth2ClientBundle`: https://github.com/knpuniversity/oauth2-client-bundle
```

Comment:
So, now we have 2 links to `KnpUOAuth2ClientBundle` as `Social Authentication` points to the same page.

`Social Authentication` points to **Authenticating with Guard**, and then we have **Check `KnpUOAuth2ClientBundle`_ if you want a Guard solution**, does it make any more sense?

Reply:
@skmedix you're right I didn't notice that `Social Authentication` points to `KnpUOAuth2ClientBundle` my mistake. I will close this PR.

Thanks you!